### ISSUE TITLE
Link to metric hub documentation from Nimbus UI

### ISSUE THREAD

**Issue by jaredlockhart** [mozilla/experimenter#8210]
There are now nice auto generated docs for each outcome in metric hub, example:

https://mozilla.github.io/metric-hub/outcomes/firefox_desktop/picture_in_picture/

Let's link to there from the Nimbus UI where ever outcomes are mentioned by ID

**Comment by kessio** [comment]
Hello @yashikakhurana I'm an Outreachy intern and would love to contribute to this issue, Thank you.

**Comment by yashikakhurana** [comment]
@kessio sure I can assign it to you, just wanted to make sure that you can access this [link ](https://mozilla.github.io/metric-hub/outcomes/firefox_desktop/picture_in_picture/)

**Comment by Avi-88** [comment]
Hi @yashikakhurana , if this issue is still available I'd like to take it up. Also could you please  give me a little more context on how to approach this issue , thanks

**Comment by Sprof22** [comment]
@yashikakhurana I am an Outreachy applicant, can I be assigned to this issue

**Comment by kessio** [comment]
@yashikakhurana yes I can access the link, thanks. Let me get started on the issue.

**Comment by yashikakhurana** [comment]
@kessio  if you are not working, I have to remove your assignment, let me know otherwise tomorrow I will remove your assignment as other applicants are waiting to get assigned, kindly submit a PR.

**Comment by Avi-88** [comment]
Hi @yashikakhurana ma'am, I'd like to work on this. If possible could you please assign it to me, thanks

**Comment by yashikakhurana** [comment]
@Avi-88 yes that's correct
